eligible/eligible.go

```diff
@@ -155,6 +147,34 @@ func clusters(group grp.InstanceGroup, cloudProvider deploy.CloudProvider, exs [
 	return result, nil
 }
 
+// regions returns list of candidate regions for termination given app config and where cluster is deployed
+func regions(group grp.InstanceGroup, deployedRegions []deploy.RegionName) []deploy.RegionName {
+	region, ok := group.Region()
+	if ok {
+		return regionsWhenAppSpecifiesSingleRegion(region, deployedRegions)
```

Comment:
What happens if an app has the same cluster in 2 out of 3 regions?  This looks like we either get single region or all regions, but nothing in between.  But I could be just mis-reading...not very familiar with Go.

Reply:
The code in lines 151-154 is to handle the case where the termination is scoped to a specific region. 

The `regions` function is always invoked in the context of a specific cluster, and a precondition of the `regions` function is that the `deployedRegions` variable only contains regions where the cluster is deployed. I added a test in 9fcb504 to verify that the two-region case is handled properly.

I'll add metrics in a future PR, because I'd like to try incorporating the [spectator-go] library for that.

[spectator-go]: https://github.com/netflix/spectator-go